Container/CSS/avatar.css

```diff
@@ -0,0 +1,135 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: content-box;
+    font-size: 1rem;
+}
+
+.avatar-round {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.avatar-rounded_xS {
+    height: 3rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_S {
+    height: 3.5rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_M {
+    height: 4rem;
+    border-radius: 50%;
+}
+
+.avatar-rounded_L {
+    height: 4.5rem;
+    border-radius: 50%;
+}
+
+.avatar-square {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.avatar-square_xS {
+    height: 3rem;
+    border-radius: 25%;
+}
+
+.avatar-square_S {
+    height: 3.5rem;
+    border-radius: 25%;
+}
+
+.avatar-square_M {
+    height: 4rem;
+    border-radius: 25%;
+}
+
+.avatar-square_L {
+    height: 4.5rem;
+    border-radius: 25%;
+}
+
+.avatar-badge-sqaure {
+    display: flex;
+    gap: 1rem;
+}
+
+.avatar-badge_xS,
+.avatar-badge_S,
+.avatar-badge_L,
+.avatar-badge_M {
+    position: relative;
+    display: inline-block;
+}
+
+.avatar-badge-sqaure p {
+    position: absolute;
+    height: 0.8rem;
+    width: 0.8rem;
+    background: chartreuse;
+    bottom: 2%;
+    right: 0;
+    border-radius: 50%;
+}
+
+.text-avatar {
+    display: flex;
+    gap: 1rem;
+    align-items: center;
+}
+
+.text-avatar_xS {
+    height: 3rem;
+    width: 3rem;
+    border-radius: 50%;
+    background: rgb(44, 42, 42);
+    text-align: center;
+    color: white;
```

Comment:
Suggestion:
Please use hexcodes for consistency.

Reply:
resolved